R/split.R

```diff
@@ -1,13 +1,20 @@
 #' Split up a string into pieces
 #'
 #' @description
-#' These functions differ primarily in their input and output types:
+#' This family of functions provide various ways of splitting a string up
+#' into pieces. There are two functions that return character vectors:
```

Comment:
```suggestion
#' into pieces. These two functions return a character vector:
```

This is a nitpick, but I've started to think that using singular in these settings is much clearer.